tests/test_element.py

```diff
@@ -34,37 +36,37 @@ def assert_classes(classes: str) -> None:
 
 def test_style_parsing(nicegui_reset_globals):
     # pylint: disable=protected-access
```

Comment:
should this be removed (same for `test_props_parsing`?

also, would it be better to use test parameterization here, instead of having multiple unrelated `assert` statements together?

Reply:
Thanks for the suggestions, @marcuslimdw!
Removing the pylint markers is a good idea. They are not needed anymore.
And even though I think test parametrization is a bit a matter of taste, you and claude.ai convinced me that there are more pros than cons for following this approach.